setup.py

```diff
@@ -25,7 +25,6 @@
         'Markdown<3.0.0,>=2.6.7',
         'PyYAML<6.0,>=5.1.2',
         'inflection<1.0.0,>=0.3.1',
-        'digitalmarketplace-utils<49.0.0,>=44.0.1',
```

Comment:
Does this library not require digitalmarketplace-utils at all?

Reply:
It uses the date formatters (and the `DMSandboxedEnvironment` for handling `TemplateField` types).

Reply:
I think this should be

```suggestion
        'digitalmarketplace-utils>=44.0.1',
```